.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/...
```

Comment:
Here you need to specify just `/.idea` .  Syntax with three dots working only with golang tools like `go build`, `go lint`, `go vet` etc..